lib/piper/permissions/piper_rule_parser.yrl

```diff
@@ -42,6 +42,8 @@ command_selector ->
                                                        ?AST("BinaryExpr"):new('$4', [{left, Lhs},
                                                                                      {right, '$5'}]).
 permission_selector ->
+  permission_criterion : '$1'.
```

Comment:
Does this mean that you can define a rule without the `must have` that we previously required? If that's not the intention, we might want to move `allow` to it's own operator other than `permission_criteria`.